app/models/contact.rb

```diff
@@ -1,17 +1,12 @@
 class Contact < ApplicationRecord
-  RELATIONSHIP_TYPES = %w[friend family colleague]
+  RELATIONSHIP_TYPES = %i[friend family colleague enemy].freeze
 
   belongs_to :user
-
   has_many :interactions, dependent: :destroy
 
-  validates :user_id, presence: true
-  validates :name, :relationship,  presence: true
-  validates :name, :relationship, :email, :phone_number, :address, :social_media_handle, presence: true
+  validates :name, :relationship, :user_id, presence: true
   validates :email, format: { with: URI::MailTo::EMAIL_REGEXP, message: 'Must be a valid email address 📪' }
-  validates :relationship, inclusion: { in: %w[friend family colleague], message: "%{value} is not a valid relationship" }
-  
-  def self.relationship_options
-    [:friend, :family, :colleague]
-  end
+  validates :relationship, inclusion: {
+    in: RELATIONSHIP_TYPES.map(&:to_s), message: "%<value>s is not a valid relationship"
```

Comment:
If you revert back to `%w` in the constant, we don't need the `.map(&:to_s)`.  Not sure if rubocop flagged it.  
```suggestion
    in: RELATIONSHIP_TYPES, message: "%<value> is not a valid relationship"
```

Reply:
Good point, implemented this! ☀️